src/main/java/io/anserini/server/Controller.java

```diff
@@ -52,4 +59,34 @@ public Map<String, Object> search(@PathVariable(value = "index", required = fals
     return queryMap;
   }
 
+  @RequestMapping(method = RequestMethod.GET, path = "/isCached/{index}")
```

Comment:
See https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5

What's the best practice for a "status endpoint"?